src/app/profile/cleanup/cleanup.component.html

```diff
@@ -9,7 +9,7 @@ <h3 *ngIf="cleanupStatus === 'notstarted'">
         <strong>You are about to erase all of your OpenShift.io spaces</strong>, reverting your OpenShift.io environment to it's default state.
```

Comment:
There is a separate verbiage pass that is ongoing that might result in some other changes.  This layout matches the design as coded.